docs/changehistory/NextVersion.md

```diff
@@ -3,3 +3,11 @@ publish: false
 ---
 # NextVersion
 
+Table of contents:
+
+- [Quantity](#quantity)
+
+## Quantity
+
+- Add support for 'Ratio' format type (e.g. "1:2")
```

Comment:
Can you add an example of formatting a ratio here? You can denote outside the code snippet that you're assuming a UnitsProvider (like the SchemaUnitsProvider) has been registered or etc. Might be good to then put the example in either ec-format or QuantityFomratting.md